tfe/resource_tfe_policy_set.go

```diff
@@ -0,0 +1,300 @@
+package tfe
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceTFEPolicySet() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceTFEPolicySetCreate,
+		Read:   resourceTFEPolicySetRead,
+		Update: resourceTFEPolicySetUpdate,
+		Delete: resourceTFEPolicySetDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceTFEPolicySetImporter,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
This should also be `Computed: true` as we read back the description in the read function. So if the description is set in the web UI it will be "computed" (aka updated) here as well.